lib/perl/centreon/script/centcore.pm

```diff
@@ -336,8 +336,7 @@ sub startWorker($) {
     my $username = $data->{'contact_alias'};
     my $passwordEnc = $data->{'contact_passwd'};
 
-    my $cmdexec = "php $self->{centreonDir}/bin/centreon -u $username -p $passwordEnc -s -o CentreonWorker -a processQueue >> /var/log/centreon/worker.log";
-
+    my $cmdexec = "php72 $self->{centreonDir}/bin/centreon -u $username -p $passwordEnc -w -o CentreonWorker -a processQueue >> /var/log/centreon/worker.log";
```

Comment:
bin/centreon must ahve the right shebang. So don't use explicitly your php binary